kale/utils/chemchar_label.py

```diff
@@ -0,0 +1,82 @@
+"""
+Functions for labeling and encoding chemical characters like Compound SMILES and atom string, refer to
+https://github.com/hkmztrk/DeepDTA and https://github.com/thinng/GraphDTA.
+"""
```

Comment:
I though I have made a comment about this but it does not seem to be there.

I am wondering whether this `chemchar_label.py` module would be better to be put under `prepdata` or `utils`. Under `prepdata`, we have `tensor_reshape`, `image_transform` (mean removal etc). `utils` should be reserved for those not fitting into the other modules, such as print or logging. If needed, we could discuss this issue over skype.